roles/sharded-queue-router.lua

```diff
@@ -0,0 +1,67 @@
+local config = require('sharded_queue.router.config')
+local is_metrics_supported = require('sharded_queue.metrics').is_supported
+local metrics = require('sharded_queue.router.metrics')
+local roles = require('sharded_queue.roles')
+local utils = require('sharded_queue.utils')
+local queue = require('sharded_queue.router.queue')
+
+local role_name = "roles.sharded-queue-router"
+
+local function validate(conf)
+    conf = conf or {}
+    if not roles.is_sharding_role_enabled('router') then
```

Comment:
I think it's better to also have an assert like this

https://github.com/tarantool/crud/blob/99315a53ef75056ac057b63f5ce1bbadef6a1418/roles/crud-router.lua#L11-L15

in case one decides to run roles on older Tarantool 3. Otherwise they'll get some internal error on `config:get`.

Reply:
Thank you, updated.